src/helpers/Task.js

```diff
@@ -118,13 +124,46 @@ export default class Task {
     tutorialDescription:
       "Image enhancement models improve the resolution of an image, making it crisper and clearer.",
   });
+  static style_transfer = new Task({
```

Comment:
@ShivanshShalabh Here is the example of the `inputs` array that we were looking at on the zoom call